playbooks/templates/rax-maas/run_plugin_in_venv.sh.j2

```diff
@@ -4,6 +4,10 @@ if [[ -f {{ maas_openrc | default('~/openrc') }} ]]; then
   source {{ maas_openrc | default('~/openrc') }}
 fi
 
+if [[ -f /home/stack/stackrc ]]; then
```

Comment:
This will only exist on the director node and it should not be pertinent to overcloud which plugins are targeting.

Reply:
this is resolved